brainscore_language/benchmarks/syntaxgym/sg_prediction.py

```diff
@@ -0,0 +1,234 @@
+from typing import Union, Optional as TOptional, List as TList
+from pyparsing import *
+import numpy as np
+
```

Comment:
are the contents of this file copied from the SyntaxGym repository? If yes, could also leave them there and import here.

Reply:
Indirectly yes.  This is a subset of the original SG file.